### PR TITLE
DCC Ticketing: navigation to Consent I

### PR DESCRIPTION
**Pull request by AlexanderAlferov**
## Whats new
Navigation to from UQS to Dcc Ticketing Consent I screen

## Jira
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10644) 
[Story](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10194)

## How to test

1. Use cwa cli command: cwa ti gen -o
2. Scan QR Code
3. Check navigation to Consent I screen
4. Check that fields provider and subject are set

### CODE REVIEW

**Review by BMItr — DISMISSED** [review]
tested on oneplus 8 pro device.
lgtm 👌

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/consent/one/DccTicketingConsentOneFragment.kt [hunk 1]
```diff
@@ -53,15 +71,29 @@ class DccTicketingConsentOneFragment : Fragment(R.layout.fragment_dcc_ticketing_
...
+            fragment = this,
+            negativeButtonAction = { popBackStack() }
+        )
+    }
+
+    companion object {
+        fun uri(
+            transactionContextIdentifier: String
+        ): Uri {
+            val encodedCertId = URLEncoder.encode(transactionContextIdentifier, "UTF-8")
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/consent/one/DccTicketingConsentOneFragment.kt:95]: Please rename it ? it is not certificate Id

**Review by mtwalli — DISMISSED** [review]
LGTM, one minor comment

**Comment by mtwalli** [comment]
1- Values such as `booking provider` should be in `""`  Look Figma 
Screenshot 2021-11-22 at 20 43 02

2- Weird transition when screen is opened , fix is to put `android:background="@color/background"` to root layout in xml 

[Transition](https://user-images.githubusercontent.com/25054729/142925339-7da79e6a-7176-412e-b72b-79f4dab3ad3e.mp4)

**Review by mtwalli — APPROVED** [review]
Thanks 🙏

**Review by BMItr — APPROVED** [review]

**Pull request merged**